guide/contribute/propose-a-change.md

```diff
@@ -174,16 +174,54 @@ After changes have been published on your copy, it's time to announce them in th
    height = 400
 %}
 
-#### Applying additional changes
+#### Request a review
 
-Your proposed changes need to undergo a [review process]({{ '/guide/contribute/review/' | relative_url }}). During the review of your pull request, you may be asked to make certain changes. To do that, simply make sure that you're on the same branch as the pull request you've submitted. [Apply and push](#applying-changes) your changes.
+Your proposed changes need to undergo a [review process]({{ '/guide/contribute/review/' | relative_url }}). It often consists of several rounds of several rounds of feedback, for example:
+
+1. **Structural:** Reviewers look at how the new content fits in with the existing content in the guide. They check if it's generally appropriate, complete and accurate, and if it reads well. It gives you confidence that you are working in the right direction.
```

Comment:
I think the content reading well belongs in the Refinement stage, or even Polish stage.

```suggestion
1. **Structural:** Reviewers look at how the new content fits in with the existing content in the guide. They check if it's generally appropriate, complete and accurate. It gives you confidence that you are working in the right direction.
```